Description: The filechooser tests are not very chroot-friendly, as they
 need the gtk schemas to be compiled and installed, so skip them for now.
Author: Emilio Pozuelo Monfort <pochu@debian.org>

--- a/gtk/tests/Makefile.am
+++ b/gtk/tests/Makefile.am
@@ -69,9 +69,9 @@
 #crossingevents_SOURCES		 = crossingevents.c
 #crossingevents_LDADD		 = $(progs_ldadd)
 
-TEST_PROGS			+= filechooser
-filechooser_SOURCES		 = filechooser.c
-filechooser_LDADD		 = $(progs_ldadd)
+#TEST_PROGS			+= filechooser
+#filechooser_SOURCES		 = filechooser.c
+#filechooser_LDADD		 = $(progs_ldadd)
 
 TEST_PROGS			+= builder
 builder_SOURCES			 = builder.c
